.github/workflows/build.yml

```diff
@@ -77,7 +80,10 @@ jobs:
         with:
           key: ${{ matrix.target_os }}-${{ matrix.arch }}
       - name: Build libtelio
-        run: python3 ci/build_libtelio.py build ${{ matrix.target_os }} ${{ matrix.arch }}
+        run: |
+          python3 -m pip install -r requirements.txt || python3 -m pip install --break-system-packages -r requirements.txt
```

Comment:
what root os of this container, that you need a --breal-system-packages tag?

Reply:
Android image is based on Debian12 and requires this flag and Linux image is based on Debian11 and does no know it yet.